olive/passes/onnx/conversion.py

```diff
@@ -177,3 +187,18 @@ def _run_for_config(
 
         # save the model to the output path and return the model
         return model_proto_to_olive_model(onnx_model, output_model_path, config)
+
+
+class DeviceSpecificOnnxConversion(OnnxConversion):
+    """Convert a PyTorch model to ONNX model using torch.onnx.export by using specific hardware device."""
+
+    @staticmethod
+    def is_accelerator_agnostic(accelerator_spec: AcceleratorSpec) -> bool:
```

Comment:
What is this used for?

Reply:
it is used to decide whether the pass could be reused by other device or EPs